### DISCUSSION TITLE
Question on backing up

### DISCUSSION THREAD

**Discussion by killmasta93** [dani-garcia/vaultwarden#4340] [Q&A] [answered]
Hi
Currently wanted to use the https://github.com/tiredofit/docker-db-backup
but as it seems when installing the vaultwarden i cant seem to find the username or password of the db any ideas?

Thank you

**Comment by RealOrangeOne** [comment, marked as answer]
Are you using sqlite for the DB? That won't have a usename / password in the conventional sense, it's a file on the filesystem. I think the Vaultwarden admin shows which database engine you're using.

**Reply by killmasta93** [reply]
oh yeah just realized its sqlite, so just backing up that file is enough?

**Reply by RealOrangeOne** [reply]
Sort of, yes. Although it's not always reliable to just copy it. You'll want to do the backup properly: https://stackoverflow.com/questions/25675314/how-to-backup-sqlite-database
